### PR TITLE
Add auth0 entry for tile and documentation

### PR DESCRIPTION
**Pull request by Deborah-Digges**
### What does this PR do?

Adding the tile and documentation for the Datadog-Auth0 integration.
### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [x] Feature or bugfix has tests
- [x] Git history is clean
- [x] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: auth0/manifest.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "guid": "9308a35c-219e-4d24-ac11-af2511e5041a",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Auth0 Integration",
+  "categories": [
+    "monitoring",
```
Comment (anshumgargdd) [inline auth0/manifest.json]: please replace with. "categories": ["log collection" , "security"]

File: auth0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "display_name": "Auth0",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "Auth0",
```
Comment (anshumgargdd) [inline auth0/manifest.json]: please lowercase "name": "auth0"

File: auth0/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+	
+6. Click Save.
+7. You're done! Now when Auth0 writes the next tenant log, you'll receive a copy of that log event in Datadog with the source and service set to `auth0`.
+
+### Validation
+
+View logs in Datadog:
+
+1. Navigate to **Logs** > **Livetail**.
+2. See Auth0 logs by setting `service:auth0`.
```
Comment (anshumgargdd) [inline auth0/README.md]: typically we suggest `source:auth0`

File: auth0/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
+# Agent Check: auth0
+
+## Overview
+
+Use the Datadog-Auth0 integration to view and analyze your log events from Auth0.
+
+## Setup
+
+All configuration happens on the [Auth0 Dashboard][2]. Find details on how to set it up in the [Auth0 Documentation][3]. Datadog automatically creates a dashboard when it detects Auth0 logs.
```
Comment (anshumgargdd) [inline auth0/README.md]: We need to create a dashboard it's not create yet. We can remove the line for now.

File: auth0/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+### Validation
+
+View logs in Datadog:
+
+1. Navigate to **Logs** > **Livetail**.
+2. See Auth0 logs by setting `service:auth0`.
+
+## Data Collected
+
+### Metrics
```
Comment (anshumgargdd) [inline auth0/README.md]: we can add a heading with "### Logs" with some details on the kind of data these logs have. It doesn't need too much detail but perhaps either classes of logs being sent or important data types and a link to all data types https://auth0.com/docs/logs/references/log-event-type-codes

File: auth0/manifest.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Auth0 Integration",
+  "categories": [
+    "monitoring",
+    "alerting",
+    "security"
+  ],
```
Comment (anshumgargdd) [inline auth0/manifest.json]: please replace with. "categories": ["log collection" , "security"]

File: auth0/manifest.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+  "display_name": "Auth0",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "Auth0",
+  "metric_prefix": "auth0.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "View and analyze your Auth0 events.",
+  "guid": "9308a35c-219e-4d24-ac11-af2511e5041a",
+  "support": "core",
```
Comment (hithwen) [inline auth0/manifest.json]: It cannot be core if its not in core repo

File: auth0/README.md (hunk 4 above)
Comment (Deborah-Digges) [reply auth0/README.md]: Removed line about Dashboard

File: auth0/README.md (hunk 3 above)
Comment (Deborah-Digges) [reply auth0/README.md]: updated to `source:auth0`

File: auth0/README.md (hunk 5 above)
Comment (Deborah-Digges) [reply auth0/README.md]: Added `### Logs` section

File: auth0/manifest.json (hunk 2 above)
Comment (Deborah-Digges) [reply auth0/manifest.json]: lowercased name

File: auth0/manifest.json (hunk 7 above)
Comment (Deborah-Digges) [reply auth0/manifest.json]: Updated `support` to `contrib`

File: auth0/manifest.json (hunk 6 above)
Comment (Deborah-Digges) [reply auth0/manifest.json]: Updated categories

**Pull request merged**